src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public int getRandomBall() {
+        Random randomBall = new Random();
+        int value = randomBall.nextInt(100);
+        return value;
```

Comment:
Temporary variable 'value' is unnecessary here; you could directly return the result of `randomBall.nextInt(100)`.